docs/user-guides/deployment.md

```diff
@@ -0,0 +1,245 @@
+# Deployment
+
+There are two major ways for the deployment of the Vald cluster ways: Using the Helm command with `values.yaml` or `vald-helm-operator`.
+
+- Using Helm command
+    - Easy to deploy
+    - Need to Helm command when updating configuration
+
+- Using `vald-helm-operator`
+    - Monitoring the Vald deployments and Vald custom resource by `vald-helm-operator`
+    - Use `kubectl` command to set or update of the Vald cluster configuration
+    - Automate manage the Vald cluster based on CRD (Custom Resource Definitions)
+
+## Requirement
+
+- Helm: v3 ~
+
+If Helm or HDF5 is not installed, please install [Helm](https://helm.sh/docs/intro/install).
+
+<details><summary>Installation command for Helm</summary><br>
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+```
+
+</details>
+
+## Before Deployment
+
+Both ways need the configuration file when you need to overwrite the default configurations.
+
+Please refer to [the basic configuration document](../user-guides/configuration.md).
+
+In addition, as you need, please refer to the followings:
+- [Backup Configuration](../user-guides/backup-configuration.md)
+- [Filter Configuration](../user-guides/filter-configuration.md)
+
+## Use Helm command
+
+### Deployment
+
+When deploying with Helm command, it requires `value.yaml` to override the default values.
+
+<details><summary>Sample values YAML</summary><br>
+
+```yaml
+defaults:
+  logging:
+    level: debug
+  image:
+    tag: "v1.5.6"
```

Comment:
we should discuss about versions.
there are two way to manage document vald version.
1. use specific version like for now.
2. use latest tag

if we use 1 it would be better for user to understand stable yaml file, but it's harder for us to update document yaml version.
about 2 it would be better for us to manage document version, but for some user can't understand which versions yaml is this document.

Reply:
2 is better.
How about adding the comment like `please set the latest version, e.g. v1.5.6, instead of latest`